CMakeLists.txt

```diff
@@ -168,38 +168,22 @@ if(ASPECT_WITH_WORLD_BUILDER)
     ADD_SUBDIRECTORY("${WORLD_BUILDER_SOURCE_DIR}" ${CMAKE_BINARY_DIR}/world_builder/)
     INCLUDE_DIRECTORIES("${CMAKE_BINARY_DIR}/world_builder/include/")
   ELSE()
-    IF(WORLD_BUILDER_VERSION VERSION_LESS 0.5.0)
-      FILE(GLOB_RECURSE wb_files "${WORLD_BUILDER_SOURCE_DIR}/source/*.cc")
-    ELSE()
-      FILE(GLOB_RECURSE wb_files "${WORLD_BUILDER_SOURCE_DIR}/source/world_builder/*.cc")
-    ENDIF()
+    FILE(GLOB_RECURSE wb_files "${WORLD_BUILDER_SOURCE_DIR}/source/world_builder/*.cc")
```

Comment:
Ha, yes, we have `IF(WORLD_BUILDER_VERSION VERSION_GREATER_EQUAL 0.6.0)` above, so the `IF` was never active :-)